recipes-extended/xen/files/libxl-openxt-tweaks.patch

```diff
@@ -0,0 +1,133 @@
+################################################################################
+SHORT DESCRIPTION:
+################################################################################
+LibXL modifications specific to OpenXT
```

Comment:
Just a suggestion but it seems like several of these patches that fix misc. things could be merged.